edx_proctoring/static/proctoring/js/proctored_app.js

```diff
@@ -5,4 +5,16 @@ $(function() {
         model: new ProctoredExamModel()
     });
     proctored_exam_view.render();
+
+    // show the timer bar when the window scrolls.
+    $(window).scroll(function() {
+        if ($(this).scrollTop() > 60){
+            $(".proctored_exam_status").addClass('fixed');
+            $(".wrapper-course-material").css('margin-top', '58px');
```

Comment:
I'm wondering if this might be fragile to have an inline CSS rule here which affects the rendering of courseware styling that is defined in edx-platform. If we don't have this, what happens visually? I'm going to get this running locally to see for myself....